Выполнил практику по уроку №3.

Agodimon opened the pull request:
Переделал взаимодействие модели и логики на Rx-паттерн.

Agodimon committed:
Выполнил практику по уроку №3.
Переделал взаимодействие модели и логики на Rx-паттерн.

PeterBukhal-Geekbrains commented on app/src/main/java/com/bignerdranch/android/thirdquaterhw/presenter/UsersPresenter.kt:
```diff
@@ -38,14 +40,30 @@ class UsersPresenter(
+                    }
+            )
+
+        viewState
```
Это так же нужно переместить внутрь консюмера (в subscribe), сейчас все происходит на главном потоке но если получение пользователей будет на другои потоке обновление будет происходить раньше чем добавление пользователей.

PeterBukhal-Geekbrains reviewed (approved):
Есть замечания оставил комментарии в github, в целом хорошо.

Agodimon committed:
Исправил замечание:
"Это так же нужно переместить внутрь консюмера (в subscribe), сейчас все происходит на главном потоке но если получение пользователей будет на другои потоке обновление будет происходить раньше чем добавление пользователей."
